patches/react-native-image-picker+0.28.1.patch

```diff
@@ -0,0 +1,38 @@
+diff --git a/node_modules/react-native-image-picker/android/src/main/java/com/imagepicker/ImagePickerModule.java b/node_modules/react-native-image-picker/android/src/main/java/com/imagepicker/ImagePickerModule.java
```

Comment:
@Edigio 
I'm not sure this patch is needed.
Could you test on Android side, without any patch, if it works correctly only by removing `Platform.OS === "ios"` in **_ScanQrCodeScreen.tsx_**  on line 187?

This patch is not a code that fixes a bug but a code that changes a behavior: before it requires for WRITE and CAMERA permissions, now only for READ external storage. It means if you use this library to use the camera (it needs write permission too) this code probably crashes!

Reply:
@Undermaken it seemed incorrect to ask write permissions the archive and to access the camera when I only have to take a photo, in this scenario read permissions were sufficient. Now I have removed the patch